Assets/UI/Panels/citypanel.lua

```diff
@@ -1187,6 +1266,41 @@ end
 function OnShutdown()
     -- Cache values for hotloading...
     LuaEvents.GameDebug_AddValue("CityPanel", "isHidden", ContextPtr:IsHidden() );
+
```

Comment:
these were added here to match the Remove calls in the unmodified code.  The events that use CQUI methods instead of those in unmodified code are called out (so they are visible when doing a side-by-side compare in Notepad++, say..)
![image](https://user-images.githubusercontent.com/8787640/101239824-af14ad80-369f-11eb-9daf-36882b832bab.png)